translations.php

```diff
@@ -31,6 +32,8 @@
 			'th' => 'คุณสมบัติ',
 			'zh-hans' => '特点',
 			'zh-hant' => '特點',
+			'cs' => 'Funkce',
+
```

Comment:
Is there supposed to be an additional blank like?

Reply:
Not, should not be, sorry.